### PR TITLE
Update disabled state

### PR DESCRIPTION
**Pull request by cbelan-d2l**
* Change cursor for links
* Set color to chromium

image

Need to determine if this will have an issue with the change
https://search.d2l.dev/xref/GitHub/BrightspaceHypermediaComponents/sequences/d2l-sequence-navigator/d2l-outer-module.js?r=40c2bc48#178

### CODE REVIEW

**Comment by dlockhart** [comment]
Sorry I'd written a review but didn't "submit" it!

File: accordion-collapse.js [hunk 1]
```diff
@@ -73,6 +73,12 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-labs-accordion-collapse">
 				pointer-events: none;
 				position: absolute;
 			}
+			:host([disabled]) a{
```
[the remark this answers is not in the document]
Comment (cbelan-d2l) [reply accordion-collapse.js]: Fixed e4fd826

File: accordion-collapse.js [hunk 2]
```diff
@@ -73,6 +73,12 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-labs-accordion-collapse">
...
+				cursor: default;
+			}
+			:host([disabled]) d2l-icon{
+				color: var(--d2l-color-chromite);
+			}
```
[the remark this answers is not in the document]
Comment (cbelan-d2l) [reply accordion-collapse.js:81]: Fixed e4fd826
And this is different from `reflectToAttribute: true`?

[the remark this answers is not in the document]
Comment (dlockhart) [reply accordion-collapse.js:81]: Sorry no I'm mixed up with LitElement vs. Polymer. :) `reflectToAttribute: true` is what you want.

[the remark this answers is not in the document]
Comment (cbelan-d2l) [reply accordion-collapse.js:81]: Fixed b7b01e4

**Pull request merged**